docs/GettingStarted.md

```diff
@@ -60,6 +61,15 @@ You are also welcome to download the source code and build the generator yoursel
 
 Datahelix is under active development so expect new features and bug fixes. Please feel free to share any issues, feature requests, or ideas via the [GitHub issues page](https://github.com/finos/datahelix/issues).
 
+### Install via apt get
+If you are using Debian or a Debian-based linux distribution such as Ubuntu you can install and update the datahelix using apt-get or apt. To do this you will need to add the datahelix package repository as a source. You can do this by adding the line
+```
+deb [trusted=yes] https://apt.fury.io/datahelix/ /
```

Comment:
Do we have documentation somewhere on how to upload to this repository?

See the chocolatey documentation here: https://github.com/finos/datahelix/tree/master/chocolatey

Reply:
Happy to add this and it shouldn't take long as it's a fairly simple process. Where do you think this file should be located? The chocalatey docs are located in a chocolatey folder rather than with the others but there is no equivalent folder for apt-get

Reply:
I'd create a document in the `docs` folder and move the chocolatey document at the same time for consistency.
By all means create a subfolder under `docs` named appropriately - 'packaging'?

Reply:
Moved the chololatey doc to `docs/developer/packaging.md`, alongside a new apt-get packaging/publishing doc.